query-service-impl/src/main/java/org/hypertrace/core/query/service/multivalue/FunctionTransformation.java

```diff
@@ -8,15 +8,20 @@
 import com.google.inject.Inject;
 import io.reactivex.rxjava3.core.Single;
 import java.util.List;
+import lombok.extern.slf4j.Slf4j;
 import org.hypertrace.core.attribute.service.cachingclient.CachingAttributeClient;
 import org.hypertrace.core.attribute.service.v1.AttributeKind;
+import org.hypertrace.core.attribute.service.v1.AttributeMetadata;
 import org.hypertrace.core.query.service.api.Function;
 
+@Slf4j
 public class FunctionTransformation {
   protected static final String DISTINCT_COUNT = "DISTINCTCOUNT";
   protected static final String DISTINCT_COUNT_MV = "DISTINCTCOUNTMV";
   private static final List<AttributeKind> ARRAY_KINDS =
       List.of(TYPE_STRING_ARRAY, TYPE_BOOL_ARRAY, TYPE_DOUBLE_ARRAY, TYPE_INT64_ARRAY);
+  private static final AttributeMetadata DEFAULT_ATTRIBUTE_METADATA =
```

Comment:
Sorry I missed this, if un-used `DEFAULT_ATTRIBUTE_METADATA`, can we remove this?

Reply:
Done